socorro/signature/rules.py

```diff
@@ -725,11 +725,11 @@ def action(self, raw_crash, processed_crash, notes):
             new_sig = 'IPCError-browser | {}'
         else:
             new_sig = 'IPCError-content | {}'
-        new_sig = new_sig.format(raw_crash['ipc_channel_error'][:100])
+        new_sig = new_sig.format(drop_unicode(raw_crash['ipc_channel_error'][:100]))
 
         notes.append(
             'Signature replaced with an IPC Channel Error, '
-            'was: "{}"'.format(processed_crash['signature'])
+            'was: "{}"'.format(drop_unicode(processed_crash['signature']))
```

Comment:
I'm pretty sure you don't need to call `drop_unicode` here. Did you hit problems which required us to do this?

Reply:
I thought `processed_crash['signature']` may contain multi byte character
https://bugzilla.mozilla.org/show_bug.cgi?id=1402838#c0

```
UnicodeEncodeError: 'ascii' codec can't encode characters in position 180-183: ordinal not in range(128)
  File "socorro/signature/__init__.py", line 98, in generate
    rule.action(raw_crash, processed_crash, notes)
  File "socorro/signature/signature_utilities.py", line 674, in action
    'was: "{}"'.format(processed_crash['signature'])
```

Reply:
Oh noes! I messed up--this should be two separate bugs. The issue in the description stems from a previous rule creating a signature that had non-ASCII characters in it. That's a completely separate issue from the SignatureIPCChannelError rule creating a signature with non-ASCII characters in it.

For this PR, let's solve just the SignatureIPCChannelError issue. If we hit another problem where the signature has non-ASCII characters in it, we can use that sentry report to track down which rule is the culprit and I'll write up a bug to fix it then. It's possible it's already fixed.

Sorry about that!